Assets/Scripts/Player.cs

```diff
@@ -41,6 +72,7 @@ public override void ChangeHealth(int amount)
         if (Health == 0)
         {
             // TODO give feedback to signal game over before deleting the player
+            // TODO DONT DESTROY THE PLAYER! THIS THROWS EXCEPTIONS
```

Comment:
lol yeah it does

Reply:
Could maybe just make him stop, and rotate or something. Best if we could reset the game state but that may be too much work